### PR TITLE
AB#78 implement tests for Top Products

### PR DESCRIPTION
**Pull request by khiemkao**
Implement test scripts for US [#78](https://epampocproject.visualstudio.com/Shopee/_workitems/edit/78) Top Products section
- [AC 1] Top Products labels [#238](https://epampocproject.visualstudio.com/Shopee/_workitems/edit/238)
Screenshot 2023-01-11 at 14 32 13

### CODE REVIEW

File: src/main/java/com/epam/poc/pageObjects/homepage/TopProductsPageObject.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    }
+    public List<WebElement> getTopProducts() {
+        return getElements(driver, By.xpath(TopProductsPageUI.TOP_PRODUCT_LIST));
+    }
+
+    @Step("Click to navigation button")
+    public TopProductsPageObject clickNavigationButton(By by) {
+        waitForElementUntilClickable(driver, by);
+        hoverAndClickToElement(driver, by);
+        staticWait(1);
```
Comment (tuanna0308) [inline src/main/java/com/epam/poc/pageObjects/homepage/TopProductsPageObject.java:84]: Hi anh @khiemkao, why do you want to wait for static here?

Comment (ngothikimthaoqb) [reply src/main/java/com/epam/poc/pageObjects/homepage/TopProductsPageObject.java:84]: @tuanna0308 hi anh, I click navigation multiple times to reach the end of the carousel, so need static wait for click action effect

Comment (tuanna0308) [reply src/main/java/com/epam/poc/pageObjects/homepage/TopProductsPageObject.java:84]: okay @ngothikimthaoqb, I got it

**Review by tuanna0308 — APPROVED** [review]
I am ok with these code

**Review by LuanNguyenBao — APPROVED** [review]

**Pull request merged**